docs/notebooks/word_to_sentence_embedding.ipynb

```diff
@@ -0,0 +1,1264 @@
+     ]
+    }
+   ],
+   "source": [
+    "! ../sent2vec-master/./fasttext sent2vec -input sent2vec.txt -output my_model -dropoutK 0 -dim 200 -epoch 9 -lr 0.2 -thread 10 -bucket 100000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sent2vec\n",
+    "sent2vec_model = sent2vec.Sent2vecModel()\n",
+    "sent2vec_model.load_model('my_model.bin')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training Classic LSI model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dictionary(131056 unique tokens: [u'fawn', u'tsukino', u'woode', u'nunnery', u'sonja']...)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from gensim.corpora import Dictionary, MmCorpus\n",
+    "\n",
+    "dictionary = Dictionary(line.split() for line in open('sent2vec.txt'))\n",
+    "print dictionary\n",
+    "corpus = [dictionary.doc2bow(text) for text in sentences]\n",
+    "MmCorpus.serialize('lsi_model.mm', corpus) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gensim.models import LsiModel\n",
+    "lsi_model = LsiModel(corpus,id2word=dictionary)\n",
+    "lsi_corpus = lsi_model[corpus]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[(0, u'-0.447*\"infinity\" + -0.447*\"war\" + -0.447*\"eagerly\" + -0.447*\"waiting\" + -0.447*\"avengers\" + 0.000*\"baby\" + -0.000*\"attempted\" + -0.000*\"day\" + -0.000*\"completely\" + -0.000*\"different\"'), (1, u'-0.640*\"dog\" + -0.396*\"sample\" + -0.396*\"cat\" + -0.396*\"sentence\" + -0.245*\"day\" + -0.245*\"cute\" + 0.000*\"baby\" + -0.000*\"skipper\" + 0.000*\"attempted\" + -0.000*\"absolute\"')]\n",
+      "<gensim.interfaces.TransformedCorpus object at 0x103c44150>\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(lsi_model.print_topics(2))\n",
+    "print(lsi_corpus)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[(1, -1.8280155323464498), (3, 0.811393377776851)]\n",
+      "[(2, 1.7320508075688772)]\n",
+      "[(1, -1.1297757309528396), (3, -1.3128620634895272)]\n",
+      "[(0, -2.2360679774997907)]\n",
+      "[(4, -1.4142135623730954)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "for doc in lsi_corpus:\n",
+    "    print(doc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Implementation of mixture methods"
```

Comment:
You already have this code in `keyedvectors.py`, please use it (instead of duplication)